indexer/packages/redis/src/caches/state-filled-quantums-cache.ts

```diff
@@ -0,0 +1,38 @@
+import { RedisClient } from 'redis';
+
+import { getAsync, setexAsync } from '../helpers/redis';
+
+export const STATE_FILLED_QUANTUMS_CACHE_KEY_PREFIX: string = 'v4/state_filled_quantums/';
+export const STATE_FILLED_QUANTUMS_TTL_SECONDS: number = 300; // 5 minutes
+
+export async function updateStateFilledQuantums(
```

Comment:
can you add comments to this? Are these optimistically filled quantums or onchain confirmed filled quantums?

Reply:
It's the second. `State` is equivalent to `on-chain confirmed` when referring to values on the network.